### PR TITLE
#38 - add test page and drush command 

### PR DESCRIPTION
**Pull request by svendecabooter**

### CODE REVIEW

File: drush/encrypt.drush.inc [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+use Drupal\encrypt\Entity\EncryptionProfile;
+
+/**
+ * Implements hook_drush_command().
+ */
+function encrypt_drush_command() {
+  $items = array();
+
+  $items['encrypt'] = array(
```
Comment (nerdstein) [inline drush/encrypt.drush.inc]: I think we should have a "test profile" one that runs through a test with a random string

**Comment by nerdstein** [comment]
Reviewed, please consider a "test" command for drush and let me know your thoughts

**Comment by svendecabooter** [comment]
You mean a Drush command that will run the validate() function on a given profile, to see if the profile dependencies are met etc...?

Or you mean you can encrypt / decrypt with a test profile?

**Comment by nerdstein** [comment]
Validate, yes.

It should error if deps are not met and message why

**Comment by svendecabooter** [comment]
OK that should be fairly straight forward, and a good addition. I'll come up with some code to add this.

**Comment by nerdstein** [comment]
Looks good

**Pull request merged**